submanifests/optional.yaml

```diff
@@ -46,10 +48,10 @@ manifest:
       groups:
         - optional
     - name: tflite-micro
-      revision: 1a34dcab41e7e0e667db72d6a40999c1ec9c510c
+      revision: e1eecbee18aa8f7f0ad7e90f8b335564eb52c13f
       path: optional/modules/lib/tflite-micro
       repo-path: tflite-micro
-      remote: upstream
```

Comment:
Is it ok to do this ?

Reply:
> Is it ok to do this ?

It is not. AFAICS, it is only supposed to be a temporary measure until we decide on how to handle upstream sync for the tflite-micro module.